File: legacy/product-service/src/main/java/pl/altkom/asc/lab/micronaut/poc/product/service/infrastructure/adapters/config/PostgresDatasourceProperties.java
package pl.altkom.asc.lab.micronaut.poc.product.service.infrastructure.adapters.config;

import io.micronaut.context.annotation.ConfigurationProperties;
import io.micronaut.core.annotation.Introspected;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Introspected
@ConfigurationProperties("postgres")
public class PostgresDatasourceProperties {

    private String host = "localhost";
    private int port = 5432;
    private String database = "product";
    private String username = "product";
    private String password = "product";
    private boolean ssl = false;
}

File: legacy/documents-service/src/main/kotlin/pl/altkom/asc/lab/micronaut/poc/documents/infrastructure/config/PostgresDatasourceProperties.kt
package pl.altkom.asc.lab.micronaut.poc.documents.infrastructure.config

import io.micronaut.context.annotation.ConfigurationProperties
import io.micronaut.core.annotation.Introspected

@Introspected
@ConfigurationProperties("postgres")
class PostgresDatasourceProperties {
    var host: String = "localhost"
    var port: Int = 5432
    var database: String = "document"
    var username: String = "document"
    var password: String = "document"
    var ssl: Boolean = false
}
